Bubble Sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,12 @@
+def bubble_sort(ls):
+    for i in range(0, len(ls) - 1):
+        for j in range(0, len(ls) - 1 - i):
+            if ls[j] > ls[j + 1]:
+                tmp = ls[j]
+                ls[j] = ls[j + 1]
+                ls[j + 1] = tmp
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpur494smj/Bubble Sort/Python/bubble_sort.py
+++ b/tmp/tmpur494smj/Bubble Sort/Python/bubble_sort.py
@@ -6,6 +6,7 @@
                 ls[j] = ls[j + 1]
                 ls[j + 1] = tmp
 
+
 if __name__ == '__main__':
     arr = [2, 4, 5, 7, 8, 10]
     bubble_sort(arr)
```